Packs/PowershellPayloadResponse/Playbooks/Powershell_Payload_Response.yml

```diff
@@ -1641,37 +1354,7 @@ view: |-
       }
     }
   }
-inputs:
-- key: Endpoint ID
-  value:
-    simple: ${incident.endpointid}
-  required: false
-  description: ID of the endpoint in question.
-  playbookInputQuery:
-- key: Hostname
-  value:
-    simple: ${incident.hostname}
-  required: false
-  description: Hostname of the endpoint in question.
-  playbookInputQuery:
-- key: Email
-  value:
-    simple: ${incident.email}
-  required: false
-  description: Email attachment.
-  playbookInputQuery:
-- key: File
-  value:
-    simple: ${File}
-  required: false
-  description: Payload file that was executed.
-  playbookInputQuery:
-- key: File Path
-  value:
-    simple: ${incident.filepath}
-  required: false
-  description: File Path of the Executed Payload.
-  playbookInputQuery:
+inputs: []
```

Comment:
How was this removed? Is the playbook not requiring any input?

Reply:
See: https://github.com/demisto/content/pull/24482#issuecomment-1422710827